keras_nlp/layers/transformer_decoder.py

```diff
@@ -221,6 +236,8 @@ def get_config(self):
                 "dropout": self.dropout,
                 "activation": self.activation,
                 "layer_norm_epsilon": self.layer_norm_epsilon,
+                "kernel_initializer": initializers.serialize(self.kernel_initializer),
```

Comment:
Please use `keras.initializers` instead of `from keras import initializers`.

Sorry this is our bad, we should document is somewhere.